### DISCUSSION TITLE
Portal 1 mod: Chell's Legend test element

### DISCUSSION THREAD

**Discussion by SimpleFoxOfficial** [BEEmod/BEE2-items#4113] [Feature Suggestions]
Pads are similar to the excursion funnel in the Chell Legend mod, but if you are on its way, the time slows down for you.
![image](https://user-images.githubusercontent.com/82114463/164408287-47fbe5d4-be1b-42cc-8b96-70fd4e64ada5.png)

Also it have a mini-decoration variant
![image](https://user-images.githubusercontent.com/82114463/164408426-f40bb1f6-a24a-4e7b-b185-58dc83223dec.png)

I think it will be interesting to use it in different test chambers.

**Comment by BenVlodgi** [comment]
Cool!
What sort of puzzles can you make with it?
Does it affect objects too?

**Reply by SimpleFoxOfficial** [reply]
If you in pad way, slowing down all the world

**Reply by SimpleFoxOfficial** [reply]
Puzzles can be combined with all other items to make something very dynamic like throwing a cube on one button and having time to catch the second cube and also throwing it on the second button

**Reply by BenVlodgi** [reply]
Does the field of this object travel through portals like excursion funnels?
If the whole world slows down, I don't see much opportunity for puzzling. Its not much different from changing the game speed. To give this more purpose, some items/the player would need to be immune or change immunity to the effect.

I made a prototype a long time ago in source engine where you could speed up and slow down time at will, but you were not affected by it. I remade the prototype in Unreal Engine [Aevitas Prototype Video](https://youtu.be/iDddJUa0rN0). But it didn't have enough interesting puzzle opportunities.

**Reply by SimpleFoxOfficial** [reply]
Yes, it's working throygh portals

**Reply by SimpleFoxOfficial** [reply]
Maybe it can not affect the world, but affect player with reversed mode

**Reply by slambo311** [reply]
You could use it where you drop right into a set of turrets, which normally would require really fast reaction time to get through, but when time's slowed down, you can easily grab one turret and knock over the others. Same goes for when you have a lot of momentum and need to place portals in a certain place, along with picking up objects in mid-air and stuff.

**Comment by slambo311** [comment]
Does it have anything to tell players where the pad is (similar to the funnel's blue wavy particles)?
If not, it might be confusing for players.
Also, is there any other new test elements in Chell's Legend? Because so far, it looks great.

**Reply by SimpleFoxOfficial** [reply]
moving panels - panels that move on two/one rails to prevent the player from jumping (can be used with pads)

**Reply by SimpleFoxOfficial** [reply]
![image](https://user-images.githubusercontent.com/82114463/164463597-48fc3235-aaf3-4232-9dcd-5edbeab02ec2.png)
![image](https://user-images.githubusercontent.com/82114463/164463704-3fa0e2fa-fca7-441f-93e9-c19414c7cd04.png)

**Reply by SimpleFoxOfficial** [reply]
I also believe that the pads should be in a prominent place so that the player can understand where it is.
